dasharo-security/sort-pci-semiauto.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Library             Collections
+Library             DateTime
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+SPS001.001 Ethernet ports are in order
+    [Documentation]    This test automates the verification of port order based
+    ...    on PCIe bus numbers and checks PCIe switching.
+    Power On
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    ${pci_devices}=    Get PCIe Bus Info With MACs
+    Log    PCIe devices and their MACs: ${pci_devices}
+    Compare Interface And PCIe Bus Order    ${pci_devices}
+
+
+*** Keywords ***
+Get PCIe Bus Info With MACs
```

Comment:
The keyword no longer collects PCIe Buses, so its name has to change.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/515/commits/b319dd02a7c842be46442a66134fee681ae30d77